artemis-cli/src/test/java/org/apache/activemq/cli/test/ArtemisTest.java

```diff
@@ -233,6 +237,21 @@ public void testWebConfig() throws Exception {
       assertEquals("password2", trustPass);
    }
 
+   @Test
+   public void testStopManagementContext() throws Exception {
+      Run.setEmbedded(true);
+      File instance1 = new File(temporaryFolder.getRoot(), "instance_user");
+      System.setProperty("java.security.auth.login.config", instance1.getAbsolutePath() + "/etc/login.config");
+      Artemis.main("create", instance1.getAbsolutePath(), "--silent", "--no-autotune", "--no-web", "--no-amqp-acceptor", "--no-mqtt-acceptor", "--no-stomp-acceptor", "--no-hornetq-acceptor");
```

Comment:
Can we enhance this test (better to copy this one) and use by default web console as well?
Else Artemis process hangs as well.